animeone/lib/core/AnimeOne.dart

```diff
@@ -13,24 +14,56 @@ class AnimeOne {
     return Platform.isAndroid;
   }
 
-  Future? _invokeMethod(String method, [dynamic arguments]) {
+  Future? _invokeMethod(String method, [dynamic arguments]) async {
     if (this._isSupported()) {
-      return nativeChannel.invokeMethod(method, arguments);
+      return await nativeChannel.invokeMethod(method, arguments);
     }
 
     return null;
   }
 
   /// Restart the app
-  Future? restartApp() {
-    return this._invokeMethod('restartAnimeOne');
+  Future? restartApp() async {
+    return await this._invokeMethod('restartAnimeOne');
   }
 
   /// Popup native browser and get cookie from webview
-  Future? getAnimeOneCookie() {
-    return this._invokeMethod(
+  Future<List<String>>? _getAnimeOneCookie() async {
+    final list = await this._invokeMethod(
       'getAnimeOneCookie',
       {'link': GlobalData.requestCookieLink},
-    );
+    ) as List;
+
+    return list.map((e) => e as String).toList();
+  }
+
+  void bypassWebsiteCheck(BuildContext context) {
+    _getAnimeOneCookie()?.then((output) {
```

Comment:
Should this be here? I don't know, but this does work. Not sure if it is anti-pattern.